images/management-api/Dockerfile-nightly

```diff
@@ -11,30 +11,33 @@
 #    limitations under the License.
 #-------------------------------------------------------------------------------
 FROM graviteeio/java:8
-MAINTAINER Gravitee Team <http://gravitee.io>
+MAINTAINER Jacky Renno <jacky.renno@leansys.fr>
 
-ARG GRAVITEEIO_VERSION=0
+ARG GRAVITEEIO_VERSION=1.0.0
 
-RUN echo 'root:gravitee' | chpasswd
-RUN groupadd gravitee && useradd -s /bin/bash -m -g gravitee gravitee
-RUN echo 'gravitee:gravitee' |chpasswd
+COPY files/*.sh /usr/local/bin/
+RUN chmod u+x /usr/local/bin/*.sh
 
-USER gravitee
-WORKDIR /home/gravitee
+# Update to get support for Zip/Unzip, nc and wget
+RUN apk add --update zip unzip netcat-openbsd wget
 
-RUN wget https://bintray.com/artifact/download/gravitee-io/release/nightly/graviteeio-management-api-${GRAVITEEIO_VERSION}.zip \
-    && zip -FFv graviteeio-management-api-${GRAVITEEIO_VERSION}.zip --out graviteeio-management-api-${GRAVITEEIO_VERSION}.fixed.zip \
-    && unzip graviteeio-management-api-${GRAVITEEIO_VERSION}.fixed.zip \
-    && rm graviteeio-management-api-${GRAVITEEIO_VERSION}.fixed.zip
+RUN wget https://bintray.com/artifact/download/gravitee-io/release/nightly/graviteeio-management-api-${GRAVITEEIO_VERSION}.zip --no-check-certificate -P /tmp/ \
+    && zip -q -FFv /tmp/graviteeio-management-api-${GRAVITEEIO_VERSION}.zip --out /tmp/fixed.zip \
+    && unzip /tmp/fixed.zip -d /opt/ \
+    && rm -rf /tmp/*
 
-ENV GRAVITEEIO_HOME /home/gravitee/graviteeio-management-api-${GRAVITEEIO_VERSION}
+ENV GRAVITEEIO_HOME /opt/graviteeio-management-api-${GRAVITEEIO_VERSION}
+RUN ln -s ${GRAVITEEIO_HOME} /opt/graviteeio-management-api
 WORKDIR ${GRAVITEEIO_HOME}
 
 ENV GRAVITEEIO_MONGODB_HOST localhost
 ENV GRAVITEEIO_MONGODB_PORT 27017
 ENV GRAVITEEIO_ELASTIC_HOST localhost
 ENV GRAVITEEIO_ELASTIC_PORT 9300
 
+HEALTHCHECK --interval=30s --timeout=30s --retries=3 CMD ./usr/local/bin/healthcheck $GRAVITEEIO_ELASTIC_HOST $GRAVITEEIO_ELASTIC_PORT 5 && ./usr/local/bin/healthcheck $GRAVITEEIO_MONGODB_HOST $GRAVITEEIO_MONGODB_PORT 5
+
+ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
```

Comment:
you can remove the Entrypoint because healthcheck is now part of a dedicated docker command